tox.ini

```diff
@@ -2,9 +2,10 @@
 minversion = 2.5.0
```

Comment:
This needs to be upgraded to 3.3.0, yes?

Reply:
Indeed.